Cubical/Categories/Instances/PosetCat/Base.agda

```diff
@@ -0,0 +1,45 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Instances.PosetCat.Base where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Categories.Category
+open import Cubical.Relation.Binary.Poset
+
+open import Cubical.Categories.Instances.PosetCat.Monotone
+open import Cubical.Categories.Instances.PosetCat.MonotoneAdjoint
+
+private
+  variable
+    ℓ ℓ' : Level
+
+open Category
+
+-- Category of Posets
+PosetCat : {ℓ ℓ' : Level} → Category
+  (ℓ-max (ℓ-suc ℓ) (ℓ-suc ℓ')) (ℓ-max ℓ ℓ')
+PosetCat {ℓ} {ℓ'} = record
+  { ob = Poset ℓ ℓ'
+  ; Hom[_,_] = λ X Y → MonFun X Y
+  ; id = MonId
+  ; _⋆_ = MonComp
+  ; ⋆IdL = λ {X} {Y} f → eqMon f f refl
+  ; ⋆IdR = λ {X} {Y} f → eqMon f f refl 
+  ; ⋆Assoc = λ {X} {Y} {Z} {W} f g h → eqMon _ _ refl
+  ; isSetHom = MonFunIsSet
+  }
+
+
+-- Category of Posets w/ Adjoints
+PosetCatAdjCycl : {ℓ ℓ' : Level} → Category
+  (ℓ-max (ℓ-suc ℓ) (ℓ-suc ℓ')) (ℓ-max ℓ ℓ')
```

Comment:
You can leave these in but in general Agda can infer these so you can just use underscores if you don't feel like figuring them out.